### PR TITLE
Improve progress output for registration and deregistration operations

### PR DESCRIPTION
**Pull request by fschueller**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/160948/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 380ac291467a4cf908b4532faefe3e0ee7ce7b82 into 2ccd5b12dedd45cb1a8ff2fd7a6b324cbe1ed1d9)

File: lib/suse/connect/client.rb [hunk 1]
```diff
@@ -258,17 +257,15 @@ def remove_or_refresh_service(service)
         end
       end
 
-      def print_success_message(product, action)
+      def print_information(action)
+        server = (@config.url == 'https://scc.suse.com') ? 'SCC' : 'Registration proxy'
```
Comment (lagartoflojo) [inline lib/suse/connect/client.rb]: How about...?

```suggestion
        server = (@config.url == 'https://scc.suse.com') ? 'SUSE Customer Center' : "registration proxy #{@config.url}"
```

Comment (fschueller) [reply lib/suse/connect/client.rb]: Fixed in 6618b7b

File: features/activation.feature [hunk 2]
```diff
@@ -5,33 +5,23 @@ Feature: Test product activation
   Scenario: System registration
     Given I have a system with activated base product
 
-    Then the output should contain "Announcing system to SCC/Registration proxy ..."
+    Then the output should contain "Registering system to SCC."
```
[the remark this answers is not in the document]
Comment (fschueller) [reply features/activation.feature]: See cbca05a

File: features/activation.feature [hunk 3]
```diff
@@ -5,33 +5,23 @@ Feature: Test product activation
...
 
   @skip-sles-12
   Scenario: System registration
     Given I have a system with activated base product
 
-    Then the output should contain "Announcing system to SCC/Registration proxy ..."
-    And the output should contain "Registered SLES 15"
-    And the output should contain "To server: https://scc.suse.com"
-    And the output should contain "=========="
+    Then the output should contain "Registering system to SCC."
```
[the remark this answers is not in the document]
Comment (fschueller) [reply features/activation.feature]: See cbca05a

File: lib/suse/connect/client.rb [hunk 4]
```diff
@@ -31,12 +32,12 @@ def register!
         # Only register recommended packages for base products
         register_product_tree(show_product(product)) if product.isbase
 
-        log.info "Successfully registered system.\n".log_green.bold
+        log.info "\nSuccessfully registered system.\n".log_green.bold
```
[the remark this answers is not in the document]
Comment (fschueller) [reply lib/suse/connect/client.rb]: See cbca05a

File: spec/connect/client_spec.rb [hunk 5]
```diff
@@ -337,10 +337,24 @@
...
 
-      it 'prints message on successful activation' do
-        expect(subject).to receive(:register_product).exactly(4).times
-        expect(string_logger).to receive(:info).with("\e[1m\e[32mSuccessfully registered system.\n\e[0m\e[22m")
-        subject.register!
+      context 'with default config url' do
+        it 'prints message on successful activation' do
+          expect(subject).to receive(:register_product).exactly(4).times
+          expect(string_logger).to receive(:info).with("\e[1mRegistering system to SCC.\e[22m")
+          expect(string_logger).to receive(:info).with("\e[1m\e[32m\nSuccessfully registered system.\n\e[0m\e[22m")
```
[the remark this answers is not in the document]
Comment (fschueller) [reply spec/connect/client_spec.rb]: See cbca05a

File: spec/connect/client_spec.rb [hunk 6]
```diff
@@ -337,10 +337,24 @@
...
+        end
+      end
+
+      context 'with registration proxy url' do
+        before { config.url = 'https://rmt.mydomain' }
+
+        it 'prints message on successful activation' do
+          expect(subject).to receive(:register_product).exactly(4).times
+          expect(string_logger).to receive(:info).with("\e[1mRegistering system to Registration proxy.\e[22m")
+          expect(string_logger).to receive(:info).with("\e[1m\e[32m\nSuccessfully registered system.\n\e[0m\e[22m")
```
[the remark this answers is not in the document]
Comment (fschueller) [reply spec/connect/client_spec.rb]: See cbca05a

File: spec/connect/client_spec.rb [hunk 7]
```diff
@@ -595,7 +605,8 @@
         before { allow(System).to receive(:cleanup!).and_return(true) }
 
         it 'prints confirmation message' do
-          expect(string_logger).to receive(:info).with('Cleaning up ...')
+          expect(string_logger).to receive(:info).with("\e[1mDeregistering system from SCC.\e[22m")
```
[the remark this answers is not in the document]
Comment (fschueller) [reply spec/connect/client_spec.rb]: See cbca05a

File: spec/connect/client_spec.rb [hunk 8]
```diff
@@ -595,7 +605,8 @@
...
+          expect(string_logger).to receive(:info).with("\nCleaning up ...")
           expect(string_logger).to receive(:info).with("\e[1m\e[32mSuccessfully deregistered system.\n\e[0m\e[22m")
```
[the remark this answers is not in the document]
Comment (fschueller) [reply spec/connect/client_spec.rb]: See cbca05a

File: spec/connect/client_spec.rb [hunk 9]
```diff
@@ -690,12 +696,10 @@
         it 'logs success' do
           allow(System).to receive :remove_service
           allow(Zypper).to receive :remove_release_package
-          expect(string_logger).to receive(:info).with('Deactivating SLES HA 12 x86_64 ...')
+          expect(string_logger).to receive(:info).with("\e[1mDeregistering system from SCC.\e[22m")
```
[the remark this answers is not in the document]
Comment (fschueller) [reply spec/connect/client_spec.rb]: See cbca05a

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/SCC-Connect-Prophet/162096/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project connect on step docker-run-tests-sle12sp3. (Merged ec312a028f7c83b2f2ecaed89eaeeedea99d3d21 into 2ccd5b12dedd45cb1a8ff2fd7a6b324cbe1ed1d9)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/162102/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 735fe417bf22a664f5a5a8abc0baa7a452fda2d3 into 2ccd5b12dedd45cb1a8ff2fd7a6b324cbe1ed1d9)

**Pull request merged**